CrisisTextLine/Elmer
# decouple running jobs process from express app process, change launch commands

#### What's this PR do?
1. decouple running jobs process from express app process, change launch commands 
2. renames `app.js` to `runJobs.js`
#### How should this be manually tested?
`npm start`, 

then by checking `forever list` you can confirm that the two independent processes are running
#### Any background context you want to provide?
We did this to prevent one side of the app crashing (say, the express app accessed from CTL Online) from preventing the other side of the app (the jobs) from running.
